include/vcpkg/base/checks.h

```diff
@@ -34,6 +34,11 @@ namespace vcpkg::Checks
         msg_exit_with_message(line_info, msg::format(msg::msgErrorMessage).append(msg::format(m, args...)));
     }
 
+    [[noreturn]] inline void msg_exit_with_error(const LineInfo& line_info, const LocalizedString& s)
```

Comment:
I think this is a merge conflict; I don't see this one called, everything I see here is calling the one on line 30.